integration_tests/analytics/test_get.py

```diff
@@ -44,7 +44,6 @@ def test_get_analytics_summary(client: TestClient, test_db: Session, user_header
     assert type(data) is dict
     assert list(data.keys()) == EXPECTED_ANALYTICS_SUMMARY_KEYS
 
```

Comment:
For the integration tests might be worth checking the counts are correct given the data we have setup, no?

Reply:
Hmm, I hoped lines 47 through 49 captured this :thinking: